handbook/people-ops/travel.md

```diff
@@ -1,98 +1,166 @@
-- This is designed to allow each team to hold at least one event every six months (in addition to the company-wide events above).
-- This budget can be used for travel (airfare, trains, etc.), lodging (hotels, rental homes, etc.), events, meals and entertainment, and more.
-- Teams are expected to use discretion about what events are appropriate for team outings. Be inclusive! Contact @sourcegraph/people-ops with any questions.
-- Reserve plenty of time for just hanging out, touring around, having fun, etc. Quality time is important.
-- If one individual is a member of multiple teams, they will receive the same amount for each of their teams.
-- The budget can be used for online team bonding events if the team prefers.
+>  Spending over $400 USD per night requires additional approvals from  
+> Finance / People Ops. 
+> 
+> *Please send Finance/ People Ops an email with  your manager and
+> department VP cc’d including your proposal. In this   email, please
+> share screenshots from your research showing why your proposal would
+> be the most economical option given the surrounding  hotels. Finance
+> will reply with an approval or request to book other  accommodations.*
 
-## Individual travel
+## Modes of transportation:
 
-- Each teammate has one fully-paid trip per year to meet up with a group of teammates in another location.
-  - This is *in addition* to the other benefits mentioned on this page.
-  - The purpose of defining this as one “trip” (instead of providing a set dollar budget) is to avoid punishing teammates that live far away from other teammates, and who would therefore incur higher airfare costs.
+ - Economy class is recommended for booking all travel. *(If you need any higher tier of booking, reach out to Finance/People Ops for approval.)*
+ - Travelers are encouraged to book flights at least 30 days in advance to avoid premium airfare pricing.
+ - Book from travel providers that offer flexible cancellation policies and refundable rates, should your plans change at any time.
+ - Sourcegraph will pay for flight to and from locations. Any changes made to your itinerary to extend your trip and change your flight itinerary will be out of pocket expenses.
+ - Gas and parking would be considered reimbursable within the applicable budget.
+ - Ride-shares (Uber, Lyft, etc.) are encouraged whenever possible instead of renting a car.
+	 - If you’re in the US and would like to rent a car, we recommend checking with the following companies before booking your car rental elsewhere:
+		 - [Turo](https://turo.com/)  
+		 - [Getaround](https://www.getaround.com/)
+		 - [Zipcar](https://www.zipcar.com/)
 
+## Dining and your per diem:
 
-## Professional development and education
+   We ask you to use your best judgment when traveling and spending company money. Don’t go overboard with spending on food and drinks. Always ask for receipts and keep them organized.
```

Comment:
Could we consider a threshold below which we don't require receipts? Non-US teammates don't use Expensify, which makes receipt processing easier, and organising one's receipts for that €10 coffee and croissant at ~Charles de Gaulle~ a poorly organised airport because you haven't eaten for eight hours because ~Air France~ a poorly organised airline while you're simultaneously trying to make your connecting flight after being delayed at immigration isn't much fun.

Reply:
Great point @LawnGnome - we've adjusted to say receipts above $25 which is consistent with our policy in Expensify. Thanks for flagging this!